### PR TITLE
[8.0][ADD][web_x2many_add_button_position] Move Add an item button

### PR DESCRIPTION
**Pull request by tarteo**
This module moves the _Add an item_ button to the top of the list if attribute _editable_ is equal to _top_

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why not try to make a PR to Odoo for accepting this in core?

**Comment by tarteo** [comment]
@pedrobaeza Maybe have both? For companies with an older revision of Odoo 8.0 and don't want to update the base?

**Comment by jvOrsouw** [comment]
I really like this module. Really want it to be available in Odoo8 to prevent a lot of scrolling.

@tarteo 👍

**Comment by tarteo** [comment]
@pedrobaeza I tried to make a PR to Odoo. but they wouldn't accept it because 8.0 is already stable and end of life. https://github.com/odoo/odoo/pull/15847

**Comment by pedrobaeza** [comment]
OK, we have tried...

We can indeed go for both options now: try to merge this on master, and create this module.

**Comment by tarteo** [comment]
@pedrobaeza I already created the module it's for 8.0 (this PR)... When I've spare time I can make it for version 10 including the module for 10.

**Comment by tarteo** [comment]
@jvOrsouw Please use the GitHub review system.

**Review by jvOrsouw — APPROVED** [review]
Really nice functionality, prevents a lot of scrolling.

👍

File: web_x2many_add_button_position/templates/assets.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+<openerp>
+    <data>
+        <template id="assets_backend" inherit_id="web.assets_backend">
+            <xpath expr="." position="inside">
+                <script type="text/javascript" src="/web_x2many_add_button_position/static/src/js/web_x2many_add_button_position.js"></script>
+            </xpath>
+        </template>
+    </data>
+</openerp>
```
Comment (pedrobaeza) [inline web_x2many_add_button_position/templates/assets.xml]: Line at the end.

Comment (tarteo) [reply web_x2many_add_button_position/templates/assets.xml]: @pedrobaeza Thanks Pedro. Fixed it :)

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**